### PR TITLE
Add lint rule to check for spaces around variables

### PR DESCRIPTION
**Pull request by Vinnl**
@flodolo I don't actually know Python, so go easy on me :sweat_smile: But I figured a rule like this could be useful, given that you had to manually call out [two](https://github.com/mozilla/blurts-server/pull/2900#discussion_r1135851335) [instances](https://github.com/mozilla/blurts-server/pull/2900#discussion_r1134167049) of it.

### CODE REVIEW

**Review by flodolo — CHANGES_REQUESTED** [review]
I confess I've never looked into this, because it's not as strict as other rules, and we haven't been enforcing it.

With that said: we should expand this to all placeables (variables, but also term and message references), and the rule should be PS (for Placeable style).

Let me know if you prefer me to take over or continue this patch ;-)

**Comment by Vinnl** [comment]
@flodolo Yes, feel free to take it over! I'm sure you'll have it fixed much quicker than I would :sweat_smile:

**Review by flodolo — APPROVED** [review]

**Comment by flodolo** [comment]
Note: need to fix a bug with selectExpressions :-(

File: src/fluent_linter/linter.py [hunk 1]
```diff
@@ -406,6 +405,25 @@ def visit_ResourceComment(self, node):
...
+                    self.last_message_id,
+                    "PS01",
+                    f"Placeables should be preceded by exactly one space (`{{ {node.expression.id.name} }}`).",
+                )
+            if (node.span.end - 2) != node.expression.span.end:
+                self.add_error(
+                    node,
+                    self.last_message_id,
+                    "PS01",
+                    f"Placeables should be preceded by exactly one space (`{{ {node.expression.id.name} }}`).",
```
Comment (Vinnl) [inline src/fluent_linter/linter.py:422]: Heh, this is my mistake, but...

```suggestion
f"Placeables should be followed by exactly one space (`{{ {node.expression.id.name} }}`).",
```

**Pull request merged**